cpu/nrf51822/periph/random.c

```diff
@@ -47,6 +47,7 @@ int random_read(char *buf, unsigned int num)
 void random_poweron(void)
 {
     NRF_RNG->POWER = 1;
+    NRF_RNG->TASKS_START = 1;
```

Comment:
Sorry, i have committed to much.  This was for testing only. With unmodified random.c all tests are passing on nrf51